tests/unit/connector/test_confluent_kafka.py

```diff
@@ -268,13 +291,20 @@ def test_set_option_fails_for_unknown_and_construction_options(self):
             "consumer_no_such_option",
             "consumer_group",
             "consumer_topic",
-            "enable_auto_offset_store",
+        ]:
+            print("1", i)
```

Comment:
Do we really need this `print` statement here or was it just used for debugging?

Reply:
It was a mistake, I removed it.